lib/board.rb

```diff
@@ -0,0 +1,88 @@
+# rubocop:disable Metrics/CyclomaticComplexity,Metrics/PerceivedComplexity,Metrics/AbcSize
+
+class Board
+  attr_accessor :board
+  attr_reader :tokens
+  def initialize
+    @board = [
+      [' ', ' ', ' '],
+      [' ', ' ', ' '],
+      [' ', ' ', ' ']
+    ]
+    @tokens = 0
+  end
+
+  def display
+    "
+    #{@board[0][0]} | #{@board[0][1]} | #{@board[0][2]}
+    #{@board[1][0]} | #{@board[1][1]} | #{@board[1][2]}
+    #{@board[2][0]} | #{@board[2][1]} | #{@board[2][2]}
+    "
+  end
+
+  def validate_position(my_board, &get_input)
+    validated = false
+    until validated
+      puts 'Provide a valid position between 1 and 9 and not taken'
+      begin
+        pos = Integer(get_input.call)
+      rescue StandardError
+        puts 'Provide a valid integer'
+        retry
+      end
+      validated = true if pos < 10 && my_board.get_position(pos) == ' '
+    end
+    pos
+  end
```

Comment:
- [ ] There should be no `puts/print` statements anywhere outside of the `main.rb` file.
screenshot of the requirements 👇 
![Screen Shot 2020-09-24 at 11 18 50 PM](https://user-images.githubusercontent.com/48804258/94201461-4f8a6c80-febc-11ea-9437-f1d7faff8169.png)

Reply:
Hello Brittany! Thank you for your review.  The validate_position method was in the main.rb and the last TSE who reviewed our code told us to move it to the board class. Should we move it back to the main.rb? 
I did send you on Slack to get more clarification about this.